db-shared/src/main/java/com/splicemachine/db/iapi/reference/Property.java

```diff
@@ -58,7 +58,7 @@
      * Property that holds the client   IP Address
```

Comment:
I assume that you verified that the properties you just removed are not used anywhere in the code inherited from Derby, right?

Reply:
yes